src/Universe.cpp

```diff
@@ -1720,3 +1631,130 @@ int Lattice::getLatticeSurface(int cell, Point* point) {
+          surf->getBoundaryType() != BOUNDARY_NONE)
+        min_x = surf->getMinX(halfspace);
+    }
+  }
+
+  /* If a x-min boundary was not found, get the x-min from the bounding boxes
+   * of the cells */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter)
+    min_x = std::min(min_x, c_iter->second->getMinX());
+
+  _min_x = min_x;
+
+
+  // max x
+  double max_x = std::numeric_limits<double>::infinity();
+
+  /* Check if the universe contains a cell with an x-max boundary */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter) {
+    std::map<int, surface_halfspace*> surfs = c_iter->second->getSurfaces();
+
+    for (s_iter = surfs.begin(); s_iter != surfs.end(); ++s_iter) {
+      surf = s_iter->second->_surface;
+      halfspace = s_iter->second->_halfspace;
+      if (surf->getSurfaceType() == XPLANE && halfspace == -1 &&
+          surf->getBoundaryType() != BOUNDARY_NONE)
+        max_x = surf->getMaxX(halfspace);
+    }
+  }
+
+  /* If a x-max boundary was not found, get the x-max from the bounding boxes
+   * of the cells */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter)
+    max_x = std::max(max_x, c_iter->second->getMaxX());
+
+  _max_x = max_x;
+
+
+  // min y
+  double min_y = -std::numeric_limits<double>::infinity();
+
+  /* Check if the universe contains a cell with an y-min boundary */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter) {
+    std::map<int, surface_halfspace*> surfs = c_iter->second->getSurfaces();
+
+    for (s_iter = surfs.begin(); s_iter != surfs.end(); ++s_iter) {
+      surf = s_iter->second->_surface;
+      halfspace = s_iter->second->_halfspace;
+
+      if (surf->getSurfaceType() == YPLANE && halfspace == +1 &&
+          surf->getBoundaryType() != BOUNDARY_NONE)
+        min_y = surf->getMinY(halfspace);
+    }
+  }
+
+  /* If a y-min boundary was not found, get the y-min from the bounding boxes
+   * of the cells */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter)
+    min_y = std::min(min_y, c_iter->second->getMinY());
+
+  _min_y = min_y;
+
+  // max y
+  double max_y = std::numeric_limits<double>::infinity();
+
+  /* Check if the universe contains a cell with an y-max boundary */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter) {
+    std::map<int, surface_halfspace*>surfs = c_iter->second->getSurfaces();
+
+    for (s_iter = surfs.begin(); s_iter != surfs.end(); ++s_iter) {
+      surf = s_iter->second->_surface;
+      halfspace = s_iter->second->_halfspace;
+      if (surf->getSurfaceType() == YPLANE && halfspace == -1 &&
+          surf->getBoundaryType() != BOUNDARY_NONE)
+        max_y = surf->getMaxY(halfspace);
+    }
+  }
+
+  /* If a y-max boundary was not found, get the y-max from the bounding boxes
+   * of the cells */
+  for (c_iter = _cells.begin(); c_iter != _cells.end(); ++c_iter)
+    max_y = std::max(max_y, c_iter->second->getMaxY());
+
+  _max_y = max_y;
+
+  //min z
```

Comment:
Let's keep consistent with the comment style. as the previous comment has the /* ... */ style, we should do that here as well. Also let's make this comment more descriptive.